frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,64 @@
+import { NS } from '../utils/consts';
+
+
+// Create storage class from
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+
+// Create new pool form
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
+
+export const poolMessage = {
+  PROGRESS:
+    'The creation of an OCS storage cluster is still in progress or have failed please try again after the storage cluster is ready to use.',
+  POOL_START: 'Pool $PoolName creation in progress',
+  POOL_DUPLICATED: 'Pool "$PoolName" already exists',
+  POOL_CREATED: 'Pool $PoolName was successfully created',
+};
+
+export const storagePool = {
+  prepareStorageClassForm: (RBDProvisioner: string) => {
+    // Select provisioner
+    cy.byTestID('storage-class-provisioner-dropdown').click()
+    cy.get(`a[id="${RBDProvisioner}-link"]`).click();
```

Comment:
```suggestion
    cy.get(`#${RBDProvisioner}-link`).click();
```

Reply:
I have tried this, but it is not worked only for #openshift-storage.rbd.csi.ceph.com-link

To make it work I have to use the escaping character for period here:
    #openshift-storage\\.rbd\\.csi\\.ceph\\.com-link